### PR TITLE
chore(utils): remove is variation

### PR DESCRIPTION
**Pull request by adnasa**
#### Summary

- removes `utils/is-variation`

### CODE REVIEW

File: .changeset/thin-poems-drop.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/utils': major
```
Comment (emmenko) [inline .changeset/thin-poems-drop.md:2]: Let's not bump a major.

We assume that the utils package is only used internally.

**Review by emmenko — CHANGES_REQUESTED** [review]

File: .changeset/thin-poems-drop.md (hunk 1 above)
Comment (adnasa) [reply .changeset/thin-poems-drop.md:2]: ah.
the assumption was that it was used externally since it was an exported package.
but sure.

**Comment by adnasa** [comment]
I generated a new changeset and removed the old one.
please review

**Review by jonnybel — APPROVED** [review]

File: .changeset/strange-windows-turn.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/utils': patch
+---
+
+remove `is-variation`
```
Comment (emmenko) [inline .changeset/strange-windows-turn.md:5]: What's the reason for removing it? Simply that it's not used anywhere?

Comment (adnasa) [reply .changeset/strange-windows-turn.md:5]: exactly. there was no use for it, neither in ui-kit or mc-frontend

**Comment by adnasa** [comment]
Is there anything else on your side @emmenko ?
i'm ready to merge this.

**Review by emmenko — APPROVED** [review]

**Review by tdeekens — APPROVED** [review]

**Pull request merged**